project_scrum/views/project_project.xml

```diff
@@ -102,7 +130,7 @@
             <a name="%(project.act_project_project_2_project_task_all)d"
                 position="replace">
                 <a t-if="record.use_tasks.raw_value"
-                    name="%(project.act_project_project_2_project_task_all)d"
+                    name="%(act_project_tasks)d"
```

Comment:
Again, another complete overwrite of an action in project, when a more surgical overwrite is cleaner and safer.

Reply:
What do you mean here? Over here I change the action to an action I define. I do this because I want to use custom views and I cannot put those views in the `project.act_project_project_2_project_task_all` action because then when users view non scrum projects are going to see scrum views

Reply:
What do you mean here? Over here I change the action to an action I define. I do this because I want to use custom views and I cannot put those views in the project.act_project_project_2_project_task_all action because then when users view non scrum projects are going to see scrum views

Reply:
Missed your comment above:

> Still can identify object by the button action property, without using parent.

The thing with how the action is set there is that it uses the notation `%()d` which is resolved to an id integer. So I was not able to get a "hook" on that.

Reply:
my bad. closed.